git-clear: add page (#6614)
tldr-pages/tldr: pages/common/git-clear.md

# git clear

> > Clear a Git working directory as if it was freshly cloned with the current branch.

Reviewer: ```suggestion
> Clear a Git working directory as if it was freshly cloned with the current branch including files in `.gitignore`.
```

> > Hard reset git branch, deletes all untracked files including the ones in .gitignore.

Reviewer 2: ```suggestion
> Clear a Git working directory as if it was freshly cloned with the current branch.
```

> Part of `git-extras`.

> > More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-brv>.

Reviewer 2: ```suggestion
> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-clear>.
```

Author: Got me

> - Hard reset git branch, deletes all untracked files including the ones in .gitignore:

Reviewer 2: ```suggestion
- Reset all tracked files, and delete all untracked files even if they are included in the `.gitignore`:
```

> - Reset all tracked files, and delete all untracked files even if they are included in the `.gitignore`:

Reviewer: ```suggestion
- Reset all tracked files and delete all untracked files even if they are included in the `.gitignore`:
```
